core/src/main/kotlin/org/jetbrains/kotlinx/dataframe/api/DataColumnType.kt

```diff
@@ -52,6 +53,8 @@ public fun AnyCol.isNumber(): Boolean = isSubtypeOf<Number?>()
 
 public fun AnyCol.isBigNumber(): Boolean = isSubtypeOf<BigInteger?>() || isSubtypeOf<BigDecimal?>()
 
+public fun AnyCol.isPrimitiveNumber(): Boolean = type().withNullability(false) in primitiveNumberTypes
```

Comment:
Should this function be public? I mean, it seems like users will almost never call it, and it pollutes the scope of column extensions. If it does, please add a KDoc.

Reply:
hmm, maybe not. On the other hand, we are only allowing statistics on columns with primitive number types, so it might be helpful to actually have a simple check or ability to filter columns for primitive number types. That said, to be helpful with the compiler plugin, we'd likely need a specific `primitiveNumberCols()` function in the selection DSL... I think I'll keep it for now and I'll remove `isBigNumber`

Reply:
I see I forgot some overloads; they are still filtering for number columns instead of primitive number columns, oops, fixing them too